[debug] [2022-02-18T16:27:03.576Z] ----------------------------------------------------------------------
[debug] [2022-02-18T16:27:03.578Z] Command:       C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\firebase-tools\lib\bin\firebase.js deploy --only functions
[debug] [2022-02-18T16:27:03.578Z] CLI Version:   10.2.0
[debug] [2022-02-18T16:27:03.579Z] Platform:      win32
[debug] [2022-02-18T16:27:03.579Z] Node Version:  v14.18.1
[debug] [2022-02-18T16:27:03.583Z] Time:          Fri Feb 18 2022 11:27:03 GMT-0500 (hora de Ecuador)
[debug] [2022-02-18T16:27:03.584Z] ----------------------------------------------------------------------
[debug] 
[debug] [2022-02-18T16:27:03.654Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2022-02-18T16:27:03.655Z] > authorizing via signed-in user (pablopicon6@gmail.com)
[debug] [2022-02-18T16:27:03.655Z] [iam] checking project sgintegrador for permissions ["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","firebase.projects.get"]
[debug] [2022-02-18T16:27:03.659Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/sgintegrador:testIamPermissions [none]
[debug] [2022-02-18T16:27:03.659Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/sgintegrador:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","firebase.projects.get"]}
[debug] [2022-02-18T16:27:04.529Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/sgintegrador:testIamPermissions 200
[debug] [2022-02-18T16:27:04.529Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/sgintegrador:testIamPermissions {"permissions":["cloudfunctions.functions.create","cloudfunctions.functions.delete","cloudfunctions.functions.get","cloudfunctions.functions.list","cloudfunctions.functions.update","cloudfunctions.operations.get","firebase.projects.get"]}
[debug] [2022-02-18T16:27:04.529Z] >>> [apiv2][query] POST https://iam.googleapis.com/v1/projects/sgintegrador/serviceAccounts/sgintegrador@appspot.gserviceaccount.com:testIamPermissions [none]
[debug] [2022-02-18T16:27:04.529Z] >>> [apiv2][body] POST https://iam.googleapis.com/v1/projects/sgintegrador/serviceAccounts/sgintegrador@appspot.gserviceaccount.com:testIamPermissions {"permissions":["iam.serviceAccounts.actAs"]}
[debug] [2022-02-18T16:27:05.074Z] <<< [apiv2][status] POST https://iam.googleapis.com/v1/projects/sgintegrador/serviceAccounts/sgintegrador@appspot.gserviceaccount.com:testIamPermissions 200
[debug] [2022-02-18T16:27:05.074Z] <<< [apiv2][body] POST https://iam.googleapis.com/v1/projects/sgintegrador/serviceAccounts/sgintegrador@appspot.gserviceaccount.com:testIamPermissions {"permissions":["iam.serviceAccounts.actAs"]}
[info] 
[info] === Deploying to 'sgintegrador'...
[info] 
[info] i  deploying functions 
[debug] [2022-02-18T16:27:05.576Z] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (internal/errors.js:322:7)
    at validateString (internal/validators.js:124:11)
    at Object.join (path.js:424:7)
    at Config.path (C:\Users\Dell\AppData\Roaming\nvm\v14.18.1\node_modules\firebase-tools\lib\config.js:115:45)
    at getChildEnvironment (C:\Users\Dell\AppData\Roaming\nvm\v14.18.1\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:50:49)
    at runTargetCommands (C:\Users\Dell\AppData\Roaming\nvm\v14.18.1\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:71:14)
    at C:\Users\Dell\AppData\Roaming\nvm\v14.18.1\node_modules\firebase-tools\lib\deploy\lifecycleHooks.js:122:24
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
[error] 
[error] Error: An unexpected error has occurred.
